src/Stratis.Bitcoin/FullNode.cs

```diff
@@ -292,6 +301,8 @@ public void Dispose()
             this.logger.LogInformation("Notify application has stopped.");
             this.nodeLifetime.NotifyStopped();
 
+            this.nodeRunningLock.UnlockNodeFolder();
```

Comment:
Note that if the node crashes the lockfile has to be manually removed.

Reply:
Although the case I observed this in was a crash during startup, not normal operation.

Reply:
>Note that if the node crashes the lockfile has to be manually removed.

No, it will work actually since no other application will be having this file opened.

Reply:
Ok, I tested it with a manually created lockfile and it does work like that.